src/main/kotlin/graphics/scenery/primitives/SSBOTest.kt

```diff
@@ -0,0 +1,184 @@
+package graphics.scenery.primitives
```

Comment:
This is the completely wrong place for an example 😅 And if it's not an actual test (which seems to be the case) then it should not be named `Test` (this might actually lead to issues with auto-discovery of tests, some of my logic there just filters by class name)

Reply:
Ahh yes, at that point I wasn't sure what to do to test it. Its more a help for me to visualize the general data flow through the renderer ':D